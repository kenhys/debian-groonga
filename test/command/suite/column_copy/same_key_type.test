table_create Users TABLE_HASH_KEY ShortText
column_create Users age_text COLUMN_SCALAR Text

load --table Users
[
{"_key": "alice",   "age_text": "12"},
{"_key": "bob",     "age_text": "29"},
{"_key": "charlie", "age_text": "9"}
]

table_create Friends TABLE_PAT_KEY ShortText
column_create Friends age_uint8 COLUMN_SCALAR UInt8
column_copy Users age_text Friends age_uint8

dump --dump_schema no
