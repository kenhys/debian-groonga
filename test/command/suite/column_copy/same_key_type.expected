table_create Users TABLE_HASH_KEY ShortText
[[0,0.0,0.0],true]
column_create Users age_text COLUMN_SCALAR Text
[[0,0.0,0.0],true]
load --table Users
[
{"_key": "alice",   "age_text": "12"},
{"_key": "bob",     "age_text": "29"},
{"_key": "charlie", "age_text": "9"}
]
[[0,0.0,0.0],3]
table_create Friends TABLE_PAT_KEY ShortText
[[0,0.0,0.0],true]
column_create Friends age_uint8 COLUMN_SCALAR UInt8
[[0,0.0,0.0],true]
column_copy Users age_text Friends age_uint8
[[0,0.0,0.0],true]
dump --dump_schema no
load --table Friends
[
["_key","age_uint8"],
["alice",12],
["bob",29],
["charlie",9]
]

load --table Users
[
["_key","age_text"],
["alice","12"],
["bob","29"],
["charlie","9"]
]
